cobalt/build/configs/linux-x64x11/args.gn

```diff
@@ -1,3 +1,5 @@
+import("cobalt/build/configs/media.gn")
```

Comment:
This should be removed, and I think `media.gn` could be better named (i.e. to proprietary_codecs.gn)

Reply:
Agree with the rename ,  the import is still needed, correct? :

import("cobalt/build/configs/proprietary_codecs.gn")